packages/core/src/api/message.ts

```diff
@@ -72,16 +73,19 @@ export class MessagesAPI {
 	 * @param messageRef - The message to reply to
 	 * @param options - The options to use when replying to the message
 	 */
-	public async reply(channelId: string, messageRef: APIMessage, options: RESTPostAPIChannelMessageJSONBody | string) {
-		const messageOptions = typeof options === 'string' ? { content: options } : options;
-
+	public async reply(
+		channelId: string,
+		messageRef: { channel_id: string; id: string },
+		{ files, ...body }: RESTPostAPIChannelMessageJSONBody & { files?: RawFile[] },
```

Comment:
`RESTPostAPIChannelMessageJSONBody` includes `message_reference`, which might make this a bit awkward. Maybe something like this could work. Alternatively, just get rid of the second parameter ¯\\_(ツ)\_/¯
```suggestion
		messageRef: RESTPostAPIChannelMessageJSONBody["message_reference"],
		{ files, ...body }: Omit<RESTPostAPIChannelMessageJSONBody, "message_reference"> & { files?: RawFile[] },
```

Reply:
I'd rather we just removed this method, it's not a custom api endpoint but still something that's part of the send messages api, and if we want this to be a super thin wrapper...that would be a better move